woocommerce-gateway-ebanx/includes/class-wc-ebanx-global-gateway.php

```diff
@@ -341,13 +341,13 @@ public function init_form_fields()
 			'checkout_manager_chile_document' => array(
 				'title' => __('RUT', 'woocommerce-gateway-ebanx'),
 				'type' => 'text',
-				'class' => 'ebanx-advanced-option ebanx-checkout-manager-field always-visible',
+				'class' => 'ebanx-advanced-option ebanx-checkout-manager-field chile',
```

Comment:
Can we change the class to something like `ebanx-chile-document`? Because `chile` is so generic and can cause troubles with another plugins, etc.